draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -485,16 +490,15 @@ decapsulation procedures for this hybrid KEM.
 
 `QSF-SHA3-256-ML-KEM-768-P-256` KeyGen works as follows.
 
-<!-- TODO(caw): we need to wire-encode the keys before outputting them -->
-
 <!-- TODO: is this expanding from a decaps key seed, but maybe this should just be 'expandKeyPair` -->
 <!-- TODO: annotate with the byte sizes of the parameters in terms of Nseed, Nsk, etc -->
+
 ~~~
 def expandDecapsulationKey(sk):
-  expanded = SHAKE256(sk, 96)
+  expanded = SHAKE256(sk, 112)
   (pq_PK, pq_SK) = ML-KEM-768.KeyGen_internal(expanded[0:32], expanded[32:64])
-  trad_SK = P-256.Scalar(expanded[64:96])
-  trad_PK = P-256.ScalarMultBase(trad_SK)
+  trad_SK = P-256.Scalar(expanded[64:112]) % P-256.Order()
```

Comment:
`%` needs defining above

Reply:
We could leverage the text above about `Scalar`s and `Element`s to enforce that all instances of `Scalar`s MUST be modulo their order, otherwise they aren't real `Scalar`s

Reply:
That's a cleaner way to handle this. I'll update the textual description of Scalar to include something about mapping from bytes, and say that the bytes are reduced modulo the order, or something.